spec/models/potion_spec.rb

```diff
@@ -195,6 +195,8 @@
               strength: nil,
               duration: nil,
             )
+
+            potion.reload
```

Comment:
These needed to be here to begin with - the tests were wrong and it wasn't until they started failing due to this change that I realised it.